js/privatebin.js

```diff
@@ -340,6 +340,180 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
+        },
+
+        showImagePreview: function (element, image) {
+            element.html(
+                    $(document.createElement('img'))
+                    .attr('src', image)
+                    .attr('class', 'img-thumbnail')
+                    );
+            element.removeClass('hidden');
+        },
+
+        showVideoPreview: function (element, video, mimeType) {
+            var videoPlayer = $(document.createElement('video'))
+                    .attr('controls', 'true')
+                    .attr('autoplay', 'true')
+                    .attr('loop', 'true')
+                    .attr('class', 'img-thumbnail');
+
+            videoPlayer.append($(document.createElement('source'))
+                    .attr('type', mimeType)
+                    .attr('src', video));
+            element.html(videoPlayer);
+            element.removeClass('hidden');
+        },
+
+        showAudioPreview: function (element, audio, mimeType) {
+            var audioPlayer = $(document.createElement('audio'))
+                    .attr('controls', 'true')
+                    .attr('autoplay', 'true');
+
+            audioPlayer.append($(document.createElement('source'))
+                    .attr('type', mimeType)
+                    .attr('src', audio));
+            element.html(audioPlayer);
+            element.removeClass('hidden');
+        },
+
+        showPDFPreview: function (element, pdf) {
+            //PDFs are only displayed if the filesize is smaller than about 1MB (after base64 encoding).
+            //Bigger filesizes currently cause crashes in various browsers.
+            //See also: https://code.google.com/p/chromium/issues/detail?id=69227
+
+            //Firefox crashes with files that are about 1.5MB
+            //The performance with 1MB files is bareable
+            if (pdf.length < 1398488) {
+
+                //Fallback for browsers, that don't support the vh unit
+                var clientHeight = $(window).height();
+
+                element.html(
+                        $(document.createElement('embed'))
+                        .attr('src', pdf)
+                        .attr('type', 'application/pdf')
+                        .attr('class', 'pdfPreview')
+                        .css('height', clientHeight)
+                        );
+                element.removeClass('hidden');
+            } else {
+                controller.showError(i18n._('File too large, to display a preview. Please download the attachment.'));
+            }
+        },
+
+        addDragDropHandler: function () {
+            var fileInput = controller.fileInput;
+
+            if (fileInput.length === 0) {
+                return;
+            }
+
+            function ignoreDragDrop(e) {
+                e.stopPropagation();
+                e.preventDefault();
+            }
+
+            function drop(e) {
+                e.stopPropagation();
```

Comment:
Please just use a readable variable name like `event`. If someone wants to minify the JS they can do so, but the code here should be readable. I also think some code checkers would complain…